cmake/SSGCommon.cmake

```diff
@@ -354,44 +354,27 @@ macro(ssg_build_oval_unlinked PRODUCT)
     )
 endmacro()
 
-macro(ssg_build_cpe_dictionary PRODUCT)
-    set(SSG_CPE_DICTIONARY "${CMAKE_CURRENT_SOURCE_DIR}/input/oval/platform/${PRODUCT}-cpe-dictionary.xml")
-
+macro(ssg_build_cpe_oval PRODUCT)
     add_custom_command(
-        OUTPUT "${CMAKE_BINARY_DIR}/ssg-${PRODUCT}-cpe-dictionary.xml"
         OUTPUT "${CMAKE_BINARY_DIR}/ssg-${PRODUCT}-cpe-oval.xml"
```

Comment:
Yeah, makes sense actually. In the end I think it's easier to have a bundled cpe-dictionary.xml as we don't really need to build it in the end, do we? Also we could do the same thing with cpe-oval.xml